src/library_webgpu.js

```diff
@@ -1702,6 +1720,11 @@ var LibraryWebGPU = {
     bufferWrapper.onUnmap = [];
     var buffer = bufferWrapper.object;
 
+    // Handle the defaulting of size required by WebGPU
+    if (size === {{{ gpu.WHOLE_MAP_SIZE }}}) {
+      size = undefined;
```

Comment:
Can we use Bigint support here? (what is the usual webgpu behavior for taking care of 64bit uint?)
(`size` is `size_t size` in the header)